src/renderer/pkijs/generatePems/generateUserCertificate.ts

```diff
@@ -0,0 +1,64 @@
+import { Integer, BitString } from 'asn1js'
+import { Certificate, BasicConstraints, Extension } from 'pkijs'
+
+import { signAlg, hashAlg } from './config'
+import { loadCertificate, loadPrivateKey, loadCSR } from './common'
+
+export const createUserCert = async (rootCA, rootKey, userCsr) => {
+  // const rootCACert = 'files/root_ca.pem'
+  // const rootCAKey = 'files/root_key.pem'
+
+  const userCertificate = await generateuserCertificate({
+    issuerCert: await loadCertificate(rootCA),
+    issuerKey: await loadPrivateKey(rootKey, signAlg, hashAlg),
+    pkcs10: await loadCSR(userCsr),
+    hashAlg
+  })
+  // await dumpCertificate(userCertificate)
+
+  const userCert = userCertificate.certificate.toSchema(true).toBER(false)
+  return Buffer.from(userCert).toString('base64')
+}
+
+async function generateuserCertificate({ issuerCert, issuerKey, pkcs10, hashAlg }) {
```

Comment:
generateUserCertificate


also please everywhere add return types of functions